test/large/APIAuthorTriggerTest.ts

```diff
@@ -4,13 +4,25 @@ import {expect} from 'chai';
 import {apiHelper, KiiUser, KiiThing} from './utils/APIHelper';
 import {testApp} from './utils/TestApp';
 
+import {
+    TypedID,
+    Types,
+    CommandTriggerRequest,
+    OnboardWithVendorThingIDRequest,
+    Condition,
+    Equals,
+    StatePredicate,
+    TriggersWhen,
+    APIAuthor
+} from '../../src/ThingIFSDK'
+
 declare var require: any
 let thingIFSDK = require('../../../dist/thing-if-sdk.js');
 
 describe("Large Tests for APIAuthor Trigger APIs:", function () {
 
     let user: KiiUser;
-    let au: any;
+    let au: APIAuthor;
```

Comment:
`au` is not readable variable naming would be better some meaningful word ie: 'auth'